### PR TITLE
LPS-78840 Missing translations for Activities portlet

### PR DESCRIPTION
**Pull request by IstvanD**
Hey @robertoDiaz 

Could you please review this pull request?
It is forwarded from https://github.com/IstvanD/liferay-portal/pull/56

Thank you and best regards,
István

### CODE REVIEW

**Comment by robertoDiaz** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: portal-impl/src/content/Language.properties [hunk 1]
```diff
@@ -3880,6 +3880,7 @@ repositories-require-at-least-one-write-role=Repositories require at least one w
 repository=Repository
 repository-fields=Repository Fields
 repository-id=Repository ID
+reposted-from=Reposted from
```
Comment (robertoDiaz) [inline portal-impl/src/content/Language.properties]: Why do we have to include this here?? I've found that the same key exists in microblogs-web content/Language.properties:3883 so if we include it in the portal-impl we should remove it.

File: modules/apps/collaboration/social/social-activities-web/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -1,3 +1,4 @@
 javax.portlet.title.com_liferay_social_activities_web_portlet_SocialActivitiesPortlet=Activities
 maximum-activities-to-load-at-once=Maximum Activities to Load at Once
-see-more=See More
\ No newline at end of file
+see-more=See More
+following=Following
```
Comment (robertoDiaz) [inline modules/apps/collaboration/social/social-activities-web/src/main/resources/content/Language.properties]: The new lang key should be in alphabetic order. You should also run gradlew buildLang in this bundle in order to generate the keys (at least as automatic copies)

**Comment by IstvanD** [comment]
Hey @robertoDiaz 

Thank you for the feedback.
I'll let the author of the changes know them.

Best regards,
István

**Comment by IstvanD** [comment]
Hey @laszlopap 

Could you please make the changes according to the comments in https://github.com/robertoDiaz/liferay-portal/pull/659#pullrequestreview-110353913 
and resend the pull request?

Thank you.
István

**Comment by robertoDiaz** [comment]
Thank you @IstvanD

**Pull request closed without merging**